sklearn/model_selection/_search.py

```diff
@@ -605,20 +639,29 @@ def fit(self, X, y=None, groups=None, **fit_params):
 
         # if one choose to see train score, "out" will contain train score info
         if self.return_train_score:
-            (train_scores, test_scores, test_sample_counts, fit_time,
+            (train_score_dicts, test_score_dicts, test_sample_counts, fit_time,
              score_time) = zip(*out)
         else:
-            (test_scores, test_sample_counts, fit_time, score_time) = zip(*out)
+            (test_score_dicts, test_sample_counts, fit_time,
+             score_time) = zip(*out)
+
+        # test_score_dicts and train_score dicts are lists of dictionaries and
```

Comment:
But aggregate is probably the wrong word for the helper, as it does not aggregate. It collates, perhaps